uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/controllers/document-list.controller.js

```diff
@@ -20,7 +20,8 @@ define([
     $log.debug('Controller: DocumentListController');
 
     var vm = this;
-    var onlyPendingStatuses = ['1', '2']; // 1: "awaiting upload", 2: "awaiting approval"
+    var onlyPendingStatuses = getDocumentStatusesIds(
```

Comment:
tbh, I'd use `getDocumentStatusesIds` only when doing the actual comparison and preserve the status name in case I need to do some referencing. But this is progress! Lets keep it this way for now.